custom-resource-page/src/certificate-store.ts

```diff
@@ -0,0 +1,15 @@
+import { K8sApi} from "@k8slens/extensions";
+import { Certificate } from "./certificate";
+
+export class CertifcatesApi extends K8sApi.KubeApi<Certificate> {
+}
+export const certificatesApi = new CertifcatesApi({
+  objectConstructor: Certificate
+});
+
+export class CertificatesStore extends K8sApi.KubeObjectStore<Certificate> {
+  api = certificatesApi
+}
+
+export const certificatesStore = new CertificatesStore();
+K8sApi.apiManager.registerStore(certificatesStore);
```

Comment:
I think this has changed in the latest API
```suggestion
K8sApi.apiManager.registerStore(certificatesStore,[certificatesApi]);
```

Reply:
api parameter is optional. If not provided any, api defined in store object will be used.